spinach/brokers/redis.py

```diff
@@ -272,19 +289,33 @@ def enqueue_jobs_from_dead_broker(self, dead_broker_id: uuid.UUID) -> int:
             self._to_namespaced(ALL_BROKERS_HASH_KEY),
             self._to_namespaced(ALL_BROKERS_ZSET_KEY),
             self.namespace,
-            self._to_namespaced(NOTIFICATIONS_KEY)
+            self._to_namespaced(NOTIFICATIONS_KEY),
+            self._to_namespaced(MAX_CONCURRENCY_KEY),
+            self._to_namespaced(CURRENT_CONCURRENCY_KEY),
         )
 
     def register_periodic_tasks(self, tasks: Iterable[Task]):
         """Register tasks that need to be scheduled periodically."""
-        tasks = [task.serialize() for task in tasks]
+        _tasks = [task.serialize() for task in tasks]
```

Comment:
Could you also change the `len(tasks)` to `len(_tasks)` in case someone changes the list comprehension without checking the code below it?